### PR TITLE
Couchdb integration

### PR DESCRIPTION
**Pull request by gautam-swaroop**
Added documentation for couchdb integration to signalfx/integrations.

### CODE REVIEW

File: collectd-couchdb/docs/counter.couchdb.couchdb.mrview.emits.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: Mrview Emits
+brief: Number of invocations of `emit' in map functions in the view server
```
Comment (asuresh4) [inline collectd-couchdb/docs/counter.couchdb.couchdb.mrview.emits.md]: @gautam-swaroop please change ` to '. ` are special characters in md and rst. This breaks the docs build.

File: collectd-couchdb/docs/counter.couchdb.couchdb.mrview.emits.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+metric_type: counter
+---
+### Mrview Emits
+
+Number of invocations of `emit' in map functions in the view server
```
Comment (asuresh4) [inline collectd-couchdb/docs/counter.couchdb.couchdb.mrview.emits.md]: same as above

**Comment by asuresh4** [comment]
@gautam-swaroop please see the comment above. LGTM otherwise.

**Comment by gautam-swaroop** [comment]
Made changes as per code review comments.
For installation, the last step is to restart collectd. This step is mandatory as all the `.conf` files are registered only once when the collectd starts. So we have to stop(kill) the collectd first(if it is already running) before registering any new plugin. `restart` command will also handle situation where the collectd is not running.

**Comment by barbara-sfx** [comment]
I didn't see that you added a step of killing collectd. If restarting it kills it before restarting, then that's fine - you don't need a separate step. (I am not familiar with what restart does.)

**Pull request merged**